tests/mockApp.jsx

```diff
@@ -36,7 +36,10 @@ const MockRedirect = props => {
 export const routes = [
 	{
 		path: '/',
-		component: 'div',
+		component: props =>
```

Comment:
these updates were just to prevent an irrelevant warning from being thrown in integration testing